bindings/nodejs/lib/types/wallet/transaction-options.ts

```diff
@@ -100,3 +100,15 @@ export interface CreateDelegationParams {
     /** The Account Address of the validator to which this output will delegate. */
     validatorAddress: AccountAddress;
 }
+
+/** Options for beginning staking. */
+export interface BeginStakingParams {
+    /** The account id which will become a validator. */
+    accountId: AccountId;
+    /** The amount of tokens to stake. */
+    stakedAmount: NumericString;
```

Comment:
Again this but why NumericString and not u64? I guess this should be a similar type than output amount and its u64?